src/components/user/LoginForm.js

```diff
@@ -17,37 +17,54 @@ const messages = defineMessages({
   password: { id: 'login.form.password' }
 });
 
-export const LoginForm = ({ handleSubmit, error, submitting, intl }) => (
-  <form onSubmit={handleSubmit}>
-    {error && <strong>{error}</strong>}
-    <div>
-      <Field
-        name="email"
-        label={intl.formatMessage(messages.email)}
-        component={Input}
-        type="email"
-      />
-    </div>
+const DisplayErrors = ({ error }) => {
+  if (!error) {
+    return null;
+  }
+  return (
     <div>
```

Comment:
As we talked you can try to change this to a `ul` so you just need to call `<DisplayErrors error={error} />`